src/core/agent/services/keriaNotificationService.ts

```diff
@@ -813,12 +813,14 @@ class KeriaNotificationService extends AgentService {
       }
       case OperationPendingRecordType.Oobi: {
         const connectionRecord = await this.connectionStorage.findById(
-          (operation.response as any).i
+          (operation.response as State).i
         );
         if (connectionRecord) {
           connectionRecord.pending = false;
-          connectionRecord.createdAt = (operation.response as any).dt;
+          connectionRecord.createdAt = new Date((operation.response as State ).dt);
           await this.connectionStorage.update(connectionRecord);
+          const alias = new URL(operation.metadata?.oobi).searchParams.get("name") ?? randomSalt();
```

Comment:
fix from the other PR needed here. Also, we need to set `createdAt` on KERIA here when calling `contacts().update()`